priv/repo/migrations/20160328115654_create_relays.exs

```diff
@@ -7,7 +7,7 @@ defmodule Cog.Repo.Migrations.AddRelays do
       add :name, :text, null: false
       add :token_digest, :text, null: false
       add :enabled, :boolean, null: false
-      add :desc, :text, null: true
```

Comment:
yeah, last time I wrote a separate migration, since that was better to me, but was asked to just modify the existing one since the code hadn't been released, so I'm trying to be consistent... :smile: